src/components/Menu/index.tsx

```diff
@@ -120,10 +120,9 @@ export function Item({children, label, style, onPress, ...rest}: ItemProps) {
       onFocus={onFocus}
       onBlur={onBlur}
       onPress={async e => {
-        await onPress(e)
-        if (!e.defaultPrevented) {
-          control?.close()
-        }
+        control?.close(() => {
```

Comment:
note this is a change in behavior — if control is undefined, `onPress` won't fire at all. is this desirable? can control be undefined?

Reply:
Should never be undefined, wonder why we're using optional chaining here

Reply:
This occurs because the initial value of the control must be [null](https://github.com/bluesky-social/social-app/blob/main/src/components/Menu/context.tsx#L7), as it's unknown. Refer to [this discussion](https://github.com/bluesky-social/social-app/pull/5849#discussion_r1838494512) for more details.